components/forms/FileInput/FileInput.tsx

```diff
@@ -52,28 +72,54 @@ export const FileInput = (props: FileInputProps): React.ReactElement => {
 
   return (
     <>
-      {meta.touched && meta.error ? <ErrorMessage>{meta.error}</ErrorMessage> : null}
-      <Field component={UploadField} onChange={_onChange} />
-    </>
-  );
-};
+      {meta.touched && meta.error ? (
+        <ErrorMessage id={`${name}_error`}>{meta.error}</ErrorMessage>
+      ) : null}
 
-/**
- * The input type="file" cannot be updated programatically, only via a user interaction
- * Therefore, the HOC Field wraps this UploadField component.
- * Formik will receive the updated values via the onChange function that is in the props
- */
-const UploadField = ({ ...props }) => {
-  return (
-    <>
-      <Field
-        className="gc-file-input"
-        name="uploader"
-        type={"file"}
-        data-testid="file"
-        accept={acceptedFileMimeTypes}
-        {...props}
-      />
+      <div className={classes} data-testid="file">
+        <div
+          id={name}
+          role="button"
+          tabIndex={0}
+          onClick={() => fileInputRef.current?.click()}
+          onKeyDown={(e) => {
+            if (e.key === "Enter" || e.key === " ") {
+              fileInputRef.current?.click();
+            }
+          }}
+          className="gc-file-input-upload-button"
+          aria-disabled={disabled}
+          aria-labelledby={`${ariaLabelledBy}`}
+          aria-describedby={`${name}_file_selected ${ariaDescribedBy} ${
+            meta.touched && meta.error ? `${name}_error` : null
+          }`}
+        >
+          <span aria-hidden={true}>Upload a File</span>
```

Comment:
Should this not be bilingual ?

Reply:
Yes it should.  In my rush to create the PR it was something I overlooked.